### ISSUE TITLE
Restore Flake8 linting on Github testing action

### ISSUE THREAD

**Issue by c-martinez** [CLARIAH/grlc#391]
GitHub testing action used to include Flake8 testing:

```yaml
    - name: Lint with flake8
      run: |
        # stop the build if there are Python syntax errors or undefined names
        flake8 src/ --count --select=E9,F63,F7,F82 --show-source --statistics
        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
        flake8 src/ --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

I removed it to stop tests from failing and be able to build python for different versions (see #389).

Once that is fixed, we should fix current linting issues and add flake8 testing back in.

**Comment by c-martinez** [comment]
Addressed by #440

**Comment by c-martinez** [comment]
Fixed by #440 -- closing
